site/layouts/_default/examples.html

```diff
@@ -89,74 +89,30 @@
       }
```

Comment:
Missing a style change just above. Or maybe a `// Boosted mod: no ...` from https://github.com/twbs/bootstrap/commit/d1636a7540ae935ddd27462333c2bdf5a6eadef0

Reply:
We have the style with the svg already. I can add a Boosted mod for this part yes